isacc_messaging/api/views.py

```diff
@@ -302,3 +304,22 @@ def deactivate_patient(patient_id):
         f"Patient {patient_id} active set to false",
         level='info'
     )
+
```

Comment:
everything below here should move to /migration/views.py - get its own blueprint, and be loaded in with the base_blueprint in `app.py`